src/routes/graph/update-grid.ts

```diff
@@ -35,6 +36,7 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
     fields: fieldsToServerUpdateSchema.example({ id: 'My grid' }),
     photo: Joi.string().allow(null).description('Should be a dataurl. Null clears the photo'),
     pinGroupHashIds: Joi.array().items(Joi.string()).description('Determines the set (and the order) of the pin groups in the grid'),
+    mapLayer: Joi.string().description('Determines a new map layer for all locations of group'),
```

Comment:
I wouldn't make a suggestion if the other descriptions were not written like this, sorry 😓 

```suggestion
    mapLayer: Joi.string().description('Determines a new map layer for all locations of the group'),
```